lib/auth_app/accounts.ex

```diff
@@ -114,23 +114,21 @@ defmodule AuthApp.Accounts do
 
   ## Examples
 
-      iex> apply_user_email(user, "valid password", %{email: ...})
+      iex> apply_user_email(user, %{email: ...})
       {:ok, %User{}}
 
-      iex> apply_user_email(user, "invalid password", %{email: ...})
+      iex> apply_user_email(user, %{email: "invalid"})
       {:error, %Ecto.Changeset{}}
 
   """
-  def apply_user_email(user, password, attrs) do
+  def apply_user_email(user, attrs) do
     user
     |> User.email_changeset(attrs)
-    |> User.validate_current_password(password)
     |> Ecto.Changeset.apply_action(:update)
   end
 
   @doc """
   Updates the user email using the given token.
-
```

Comment:
The first paragraph should always be a short sentence as that is used as summary in the docs.

Reply:
woops, that wasn't intentional 😃